lib/code_block.dart

```diff
@@ -89,21 +92,21 @@ class _CodeBlockState extends State<CodeBlock> {
         border: Border.all(
           color: widget.borderColor ?? Colors.black,
         ),
-        borderRadius: BorderRadius.all(Radius.circular(10)),
+        borderRadius: const BorderRadius.all(Radius.circular(10)),
```

Comment:
soru thought adding `const` in such cases was deprecated?

Reply:
Nope, exactly the other way round. Adding `new` statements is deprecated. As I included `flutter_lints` packages, we exactly follow the code style Flutter recommends.

Reply:
aaaah ok